core/src/main/scala/com/yahoo/maha/core/LiteralMapper.scala

```diff
@@ -181,3 +181,28 @@ class DruidLiteralMapper extends LiteralMapper {
     }
   }
 }
+
+class BigqueryLiteralMapper extends SqlLiteralMapper {
+  def getDateFormatFromGrain(grain: Grain): String = {
+    grain match {
+      case DailyGrain => "YYYY-MM-DD"
+      case HourlyGrain => "YYYY-MM-DD-HH"
+      case _ => "YYYY-MM-DD"
+    }
+  }
+
+  def toLiteral(column: Column, value: String, grainOption: Option[Grain] = None): String = {
+    val grain = grainOption.getOrElse(DailyGrain)
+    val escapedValue = getEscapedSqlString(value)
```

Comment:
This uses the ORACLE string escaping codec for ESAPI by default.  If BigQuery has it's own implementation, you may want to use it instead.  See postgres, we use the postgres driver to do it in the postgres executor component.

Reply:
Tried looking into this, but I couldn't find any specific escaping codec for BigQuery